### DISCUSSION TITLE
用不了诶

### DISCUSSION THREAD

**Discussion by 1564307973** [RayWangQvQ/BiliBiliToolPro#369] [General]
bili扫码登录[dev先行版]
## 开始执行... 2022-12-18 10:16:09

repo目录: /ql/data/repo
bili仓库目录: /ql/data/repo/RayWangQvQ_BiliBiliToolPro_develop

bili_dev_task_login.sh: line 9: dotnet: command not found

**Comment by RayWangQvQ** [comment]
dotnet装了吗。。。

**Reply by 1564307973** [reply]
![Screenshot_20221218_162445.jpg](https://user-images.githubusercontent.com/15557020/208288470-9846b129-5ae6-453f-aca9-9a79c33efe17.jpg)

装了啊，可是还是用不了
